workflow-service-sdk/README.md

```diff
@@ -0,0 +1,155 @@
+  }
+
+  dependencies {
+     implementation "dev.parodos:workflow-service-sdk:v1.0.0-SNAPSHOT"
+  }
+```
+
+### Others
+
+At first generate the JAR by executing:
+
+```shell
+mvn clean package
+```
+
+Then manually install the following JARs:
+
+* `target/workflow-service-sdk-v1.0.0-SNAPSHOT.jar`
+* `target/lib/*.jar`
+
+## Getting Started
+
+Please follow the [installation](#installation) instruction and execute the following Java code:
+
+```java
+
+// Import classes:
+import com.redhat.parodos.openapi.api.ApiClient;
+import com.redhat.parodos.openapi.api.ApiException;
+import com.redhat.parodos.openapi.api.Configuration;
+import com.redhat.parodos.openapi.api.models.*;
+import com.redhat.parodos.openapi.api.ProjectApi;
+
+public class Example {
+  public static void main(String[] args) {
+    ApiClient defaultClient = Configuration.getDefaultApiClient();
+    defaultClient.setBasePath("http://localhost:8080");
+
+    ProjectApi apiInstance = new ProjectApi(defaultClient);
+    ProjectRequestDTOGenerated projectRequestDTOGenerated = new ProjectRequestDTOGenerated(); // ProjectRequestDTOGenerated | 
+    try {
+      ProjectResponseDTOGenerated result = apiInstance.createProject(projectRequestDTOGenerated);
+      System.out.println(result);
+    } catch (ApiException e) {
+      System.err.println("Exception when calling ProjectApi#createProject");
+      System.err.println("Status code: " + e.getCode());
+      System.err.println("Reason: " + e.getResponseBody());
+      System.err.println("Response headers: " + e.getResponseHeaders());
+      e.printStackTrace();
+    }
+  }
+}
+
+```
+
+## Documentation for API Endpoints
+
+All URIs are relative to *http://localhost:8080*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*ProjectApi* | [**createProject**](docs/ProjectApi.md#createProject) | **POST** /api/v1/projects | Creates a new project
+*ProjectApi* | [**getProjectById**](docs/ProjectApi.md#getProjectById) | **GET** /api/v1/projects/{id} | Returns information about a specified project
+*ProjectApi* | [**getProjects**](docs/ProjectApi.md#getProjects) | **GET** /api/v1/projects | Returns a list of project
+*WorkflowApi* | [**execute**](docs/WorkflowApi.md#execute) | **POST** /api/v1/workflows | Executes a workflow
+*WorkflowApi* | [**getStatus**](docs/WorkflowApi.md#getStatus) | **GET** /api/v1/workflows/{id}/status | Returns a workflow status
+*WorkflowDefinitionApi* | [**getWorkFlowDefinitionById**](docs/WorkflowDefinitionApi.md#getWorkFlowDefinitionById) | **GET** /api/v1/workflowdefinitions/{id} | Returns information about a workflow definition by id
+*WorkflowDefinitionApi* | [**getWorkFlowDefinitions**](docs/WorkflowDefinitionApi.md#getWorkFlowDefinitions) | **GET** /api/v1/workflowdefinitions | Returns a list of workflow definition
+
+
+## Documentation for Models
+
+ - [ArgumentRequestDTOGenerated](docs/ArgumentRequestDTOGenerated.md)
```

Comment:
Is there an option to get rid of the suffix `DTOGenerated` ?

Reply:
I added the suffix `Generated` [here](https://github.com/redhat-developer/parodos/blob/de8579b246d6991d08f63621cc41db09a266f298/workflow-service-sdk/pom.xml#L104) I'll remove it :-)  

The `DTO` suffix is created by the tool, I'm not able to remove it.